.gitignore

```diff
@@ -23,6 +23,7 @@
 
 ## Generated if empty string is given at "Please type another file name for output:"
 .pdf
+main.pdf
```

Comment:
may seem super dumb - don't we just need *.pdf?

Reply:
Your team's PR has pdf files - I want to just keep it to main.pdf that isn't committed

Reply:
Think it's just being specific, only the main.pdf seems to be giving problems